src/app/directives/input-mask.directive.ts

```diff
@@ -0,0 +1,218 @@
+import { Directive, ElementRef, EventEmitter, HostListener, Input, OnInit, Output } from '@angular/core';
+
+const placeholders = {
+  'A': '^[a-zA-ZA-zА-яЁё]',
+  '0': '\\d'
+};
+
+const keys = {
+  'BACKSPACE': 8,
+  'LEFT': 37,
+  'RIGHT': 39,
+  'DEL': 46,
+  'ENTER': 13
+};
+
+
+interface IState {
+  value: string;
+}
+
+@Directive({
+  selector: '[mask]'
```

Comment:
It should have prefix "app" as per `lint` check

Reply:
@monika-Dianomic  IMHO we need to keep it as it is because this is not a app-level thing like a component. Directive mainly applied to layout fields used for validation.  `[mask], [validateEqual]` makes more sense then `[appMask], [appValidateEqual]`. So we need to bypass it at lint side. 
cc/ @praveen-garg

Reply:
Ok, that's fine.